roles/edpm_network_config/meta/argument_specs.yml

```diff
@@ -83,3 +83,10 @@ argument_specs:
         type: bool
         description: "Cleanup network interfaces not in network config"
         default: true
+      edpm_bootstrap_network_resolvconf_update:
```

Comment:
Keeping same name used in edpm_bootrap role for edpm_network_config, will propose new PR later based on edpm_network_config role and need changes in other datapalane adoption project also.